app/scripts/models/Connection.js

```diff
@@ -16,24 +16,16 @@ define(['backbone'], function(Backbone) {
       , hidden: false
     },
 
-    workspace : null,
     startNode: null,
     endNode: null,
 
-    initialize: function(args, options){
-
-      this.workspace = options.workspace;
-
-      // proxyconnections bind to the proxyMove event on the workspace
-      if ( args.startProxy || args.endProxy ) {
-        this.workspace.bind('proxyMove', this.proxyMove, this);
-      } else {
-
-        // bind to end nodes
-        this.startNode = this.workspace.get('nodes').get(args.startNodeId);
-        this.endNode = this.workspace.get('nodes').get(args.endNodeId);
-      }
+    initialize: function(args, options) {
 
+        if (!args.startProxy && !args.endProxy) {
+            // bind to end nodes
+            this.startNode = options.startNode;
+            this.endNode = options.endNode;
+        }
```

Comment:
there is no `this.proxyMove` method so I removed binding and passed start and end nodes instead of workspace as constructor parameters